src/Toolchain/ToolchainEnv.ts

```diff
@@ -75,7 +76,21 @@ class Env implements BuilderJob {
       return;
     }
 
-    this.workFlow.start(this.currentWorkspace);
+    const rootJobs = this.workFlow.jobs.filter(j => j.root === true);
+    if (rootJobs.length > 0) {
+      console.log('Showing password prompt');
+      showPasswordQuickInput().then(password => {
+        if (password === undefined) {
+          console.log('Password dialog canceled');
+          return;
+        }
+        console.log('Enter the password: ' + password);
+        process.env.userp = password;
+        this.workFlow.start(this.currentWorkspace);
+      });
+    } else {
+      this.workFlow.start(this.currentWorkspace);
+    }
```

Comment:
How about to remove printing logs to the console, at least printing password? I'm not sure that printing the user typed password is good to remain in main branch.

Reply:
Oh, it's my test code :) I will remove it. :)

Reply:
Other logs are came from vscode ssh extension.
You can see the same log before you enter the your remote machine password.

```
[12:52:49.543] > jyoung-linux@10.113.111.58's password: 
[12:52:49.544] Showing password prompt
[12:53:16.924] Got password response
[12:53:16.924] "install" wrote data to terminal: "********"
```